code/testapp/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    compileSdk = BuildConstants.ProjectConfig.COMPILE_SDK_VERSION
+
+    defaultConfig {
+        applicationId = "com.adobe.mobile.marketing.aep.testapp"
+        minSdk = BuildConstants.ProjectConfig.MIN_SDK_VERSION
+        targetSdk = BuildConstants.ProjectConfig.TARGET_SDK_VERSION
+        versionCode = BuildConstants.ProjectConfig.VERSION_CODE
+        versionName = BuildConstants.ProjectConfig.VERSION_NAME
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables.useSupportLibrary = true
+    }
+
+    buildTypes {
+        getByName(BuildConstants.BuildTypes.RELEASE) {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = BuildConstants.ProjectConfig.JAVA_SOURCE_COMPATIBILITY
+        targetCompatibility = BuildConstants.ProjectConfig.JAVA_TARGET_COMPATIBILITY
+    }
+
+    kotlinOptions {
+        jvmTarget = BuildConstants.ProjectConfig.KOTLIN_JVM_TARGET
+        languageVersion = BuildConstants.ProjectConfig.KOTLIN_LANGUAGE_VERSION
+        apiVersion = BuildConstants.ProjectConfig.KOTLIN_API_VERSION
+    }
+
+    buildFeatures {
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = BuildConstants.ProjectConfig.COMPOSE_VERSION
+    }
+
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+
+    defaultConfig {
+        namespace = "com.adobe.mobile.marketing.aep.testapp"
```

Comment:
Can you change it to something like "${buildUtils.projectNamespace}.testapp" so that it is unique for all our extension test apps?

Reply:
Sorry missed this comment. Sure, will update it.